keyboards/jlw/vault35_katana/rgb.c

```diff
@@ -0,0 +1,38 @@
+#ifdef RGB_MATRIX_ENABLE
+// LED index
+// 9    10    2    3
+//         1
+//         0
+//   8 7   6   5 4
+
```

Comment:
```suggestion
#include "quantum.h"
```
`#ifdef` is not necessary and the LED index comment is incorrect.